core/src/sumeragi/fault.rs

```diff
@@ -41,10 +36,9 @@ impl FaultInjection for NoFault {
 }
 
 /// `Sumeragi` is the implementation of the consensus. This struct allows also to add fault injection for tests.
-pub struct SumeragiWithFault<G, K, F>
+pub struct SumeragiWithFault<G, F>
```

Comment:
Also somehow this is the same struct that is ran on the actual peer. We could just rename it to `Sumeragi` and keep the fault injection behind a feature flag.

Reply:
Yea I think that's probably a good idea.